services/explorer/src/index.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
+import { SnackbarProvider } from 'notistack';
```

Comment:
I didn't see a snackbar in the design. If we introduce this, we should verify with the design team and ask for some design examples.
I would also recommend this: react-toastify if we need more customization but maybe notistack can work as well. Just make sure this is agreed upon.

Reply:
I showcased this during Friday's design meeting and everyone was fine with it :)

Reply:
Perfect